lib/library_web/templates/layout/app.html.eex

```diff
@@ -8,17 +8,29 @@
     <meta name="author" content="">
 
     <title>dwyl | Library</title>
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.6.1/css/tachyons.min.css"/>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
   </head>
 
-  <body>
-    <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
-    <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
+  <body class="bg-light-grey">
+    <div class="h3 pb1 pt2">
+      <img src="/images/dwyl-heart-only-logo-white.png" class="w3-5 mt1 db center"/>
```

Comment:
There's the [phoenix `img_tag`](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Tag.html#img_tag/2) we should use for images.